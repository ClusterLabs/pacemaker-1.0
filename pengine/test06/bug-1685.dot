digraph "g" {
"prim_shared_storage:0_post_notify_promote_0 redun2" -> "shared_storage_confirmed-post_notify_promote_0" [ style = bold]
"prim_shared_storage:0_post_notify_promote_0 redun2" [ style=bold color="green" fontcolor="black"  ]
"prim_shared_storage:0_pre_notify_promote_0 redun2" -> "shared_storage_confirmed-pre_notify_promote_0" [ style = bold]
"prim_shared_storage:0_pre_notify_promote_0 redun2" [ style=bold color="green" fontcolor="black"  ]
"prim_shared_storage:0_promote_0 redun2" -> "shared_storage_promoted_0" [ style = bold]
"prim_shared_storage:0_promote_0 redun2" [ style=bold color="green" fontcolor="black"  ]
"prim_shared_storage:1_monitor_120000 redun1" [ style=bold color="green" fontcolor="black"  ]
"prim_shared_storage:1_post_notify_promote_0 redun1" -> "shared_storage_confirmed-post_notify_promote_0" [ style = bold]
"prim_shared_storage:1_post_notify_promote_0 redun1" [ style=bold color="green" fontcolor="black"  ]
"prim_shared_storage:1_pre_notify_promote_0 redun1" -> "shared_storage_confirmed-pre_notify_promote_0" [ style = bold]
"prim_shared_storage:1_pre_notify_promote_0 redun1" [ style=bold color="green" fontcolor="black"  ]
"shared_filesystem_monitor_120000 redun2" [ style=bold color="green" fontcolor="black"  ]
"shared_filesystem_start_0 redun2" -> "shared_filesystem_monitor_120000 redun2" [ style = bold]
"shared_filesystem_start_0 redun2" [ style=bold color="green" fontcolor="black"  ]
"shared_storage_confirmed-post_notify_promote_0" -> "prim_shared_storage:1_monitor_120000 redun1" [ style = bold]
"shared_storage_confirmed-post_notify_promote_0" -> "shared_filesystem_start_0 redun2" [ style = bold]
"shared_storage_confirmed-post_notify_promote_0" [ style=bold color="green" fontcolor="orange"  ]
"shared_storage_confirmed-pre_notify_promote_0" -> "shared_storage_promote_0" [ style = bold]
"shared_storage_confirmed-pre_notify_promote_0" [ style=bold color="green" fontcolor="orange"  ]
"shared_storage_post_notify_promote_0" -> "prim_shared_storage:0_post_notify_promote_0 redun2" [ style = bold]
"shared_storage_post_notify_promote_0" -> "prim_shared_storage:1_post_notify_promote_0 redun1" [ style = bold]
"shared_storage_post_notify_promote_0" -> "shared_storage_confirmed-post_notify_promote_0" [ style = bold]
"shared_storage_post_notify_promote_0" [ style=bold color="green" fontcolor="orange"  ]
"shared_storage_pre_notify_promote_0" -> "prim_shared_storage:0_pre_notify_promote_0 redun2" [ style = bold]
"shared_storage_pre_notify_promote_0" -> "prim_shared_storage:1_pre_notify_promote_0 redun1" [ style = bold]
"shared_storage_pre_notify_promote_0" -> "shared_storage_confirmed-pre_notify_promote_0" [ style = bold]
"shared_storage_pre_notify_promote_0" [ style=bold color="green" fontcolor="orange"  ]
"shared_storage_promote_0" -> "prim_shared_storage:0_promote_0 redun2" [ style = bold]
"shared_storage_promote_0" [ style=bold color="green" fontcolor="orange"  ]
"shared_storage_promoted_0" -> "shared_filesystem_start_0 redun2" [ style = bold]
"shared_storage_promoted_0" -> "shared_storage_post_notify_promote_0" [ style = bold]
"shared_storage_promoted_0" [ style=bold color="green" fontcolor="orange"  ]
}
