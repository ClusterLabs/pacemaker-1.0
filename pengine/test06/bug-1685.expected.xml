<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="13" num_updates="24" dc-uuid="1f7cf7cc-7d90-43fb-a7bd-f13fb5c203c1" have-quorum="true" remote-tls-port="0" validate-with="pacemaker-1.0">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="short-resource-names" value="true"/><nvpair id="nvpair" name="startup-fencing" value="true"/><nvpair id="nvpair" name="stonith-enabled" value="false"/><nvpair id="nvpair" name="symmetric-cluster" value="true"/><nvpair id="nvpair" name="stop-orphan-actions" value="true"/><nvpair id="nvpair" name="stonith-action" value="reboot"/><nvpair id="nvpair" name="is-managed-default" value="true"/><nvpair id="nvpair" name="stop-orphan-resources" value="true"/><nvpair id="nvpair" name="no-quorum-policy" value="stop"/><nvpair id="nvpair" name="default-migration-threshold" value="-100"/><nvpair id="nvpair" name="probe_complete" value="true"/><nvpair id="nvpair" name="transition-idle-timeout" value="60s"/><nvpair id="nvpair" name="default-resource-stickiness" value="800"/></cluster_property_set>
     </crm_config>
     <nodes>
       <node id="1f7cf7cc-7d90-43fb-a7bd-f13fb5c203c1" uname="redun2" type="normal"/>
       <node id="cd34ed93-fa5d-4092-a618-ac6349351d13" uname="redun1" type="normal"/>
     </nodes>
     <resources>
       <master id="shared_storage"><meta_attributes id="master_slave-shared_storage.meta"/><meta_attributes id="meta_attributes"><nvpair id="nvpair" name="master-node-max" value="1"/><nvpair id="nvpair" name="master-max" value="1"/><nvpair id="nvpair" name="notify" value="yes"/><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/><nvpair id="nvpair" name="globally-unique" value="false"/></meta_attributes><primitive id="prim_shared_storage" class="ocf" provider="heartbeat" type="drbd"><meta_attributes id="primitive-prim_shared_storage.meta"/>
           <operations>
             <op name="monitor" interval="120s" id="shared_storage_op_monitor" timeout="60s"/>
           </operations>
           <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="drbd_resource" value="sip-shared"/></instance_attributes>
         </primitive></master>
       <primitive id="shared_filesystem" class="ocf" provider="heartbeat" type="Filesystem"><meta_attributes id="primitive-shared_filesystem.meta"/>
         <operations>
           <op name="monitor" interval="120s" id="shared_filesystem_op_monitor" timeout="60s"/>
         </operations>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="device" value="/dev/drbd0"/><nvpair id="nvpair" name="directory" value="/shared"/><nvpair id="nvpair" name="fstype" value="reiserfs"/></instance_attributes>
       </primitive>
     </resources>
     <constraints>
       <rsc_colocation id="colocation_shared_filesystem_shared_storage-master" rsc="shared_filesystem" with-rsc="shared_storage" with-rsc-role="Master" score="INFINITY"/>
       <rsc_order id="order_shared_filesystem_after_shared_storage-promote" first="shared_storage" then="shared_filesystem" then-action="start" first-action="promote" score="INFINITY"/>
     </constraints>
   </configuration>
   <status>
     <node_state id="1f7cf7cc-7d90-43fb-a7bd-f13fb5c203c1" uname="redun2" crmd="online" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="lrm.auto-1">
         <lrm_resources id="lrm_resources">
           <lrm_resource id="prim_shared_storage:0" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="prim_shared_storage-0_monitor_0" operation="monitor" transition-key="3:0:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:7;3:0:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="2" crm_feature_set="1.0.9" rc-code="7" op-status="0" interval="0" op-digest="f05e197b6b37f94c2a3977cc41e66556"/>
             <lrm_rsc_op id="prim_shared_storage-0_start_0" operation="start" transition-key="4:2:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:0;4:2:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="5" crm_feature_set="1.0.9" rc-code="0" op-status="0" interval="0" op-digest="f05e197b6b37f94c2a3977cc41e66556"/>
             <lrm_rsc_op id="prim_shared_storage-0_post_notify_start_0" operation="notify" transition-key="39:2:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:0;39:2:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="6" crm_feature_set="1.0.9" rc-code="0" op-status="0" interval="0" op-digest="f05e197b6b37f94c2a3977cc41e66556"/>
           </lrm_resource>
           <lrm_resource id="shared_filesystem" type="Filesystem" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="shared_filesystem_monitor_0" operation="monitor" transition-key="4:0:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:7;4:0:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="3" crm_feature_set="1.0.9" rc-code="7" op-status="0" interval="0" op-digest="5858fd8b1fcc9107c71c24186f95f937"/>
           </lrm_resource>
           <lrm_resource id="prim_shared_storage:1" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="prim_shared_storage-1_monitor_0" operation="monitor" transition-key="3:1:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:7;3:1:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="4" crm_feature_set="1.0.9" rc-code="7" op-status="0" interval="0" op-digest="f05e197b6b37f94c2a3977cc41e66556"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="transient_attributes.auto-1">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="master-prim_shared_storage:0" value="75"/></instance_attributes>
       </transient_attributes>
     </node_state>
     <node_state id="cd34ed93-fa5d-4092-a618-ac6349351d13" uname="redun1" crmd="online" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="cd34ed93-fa5d-4092-a618-ac6349351d13">
         <lrm_resources id="lrm_resources">
           <lrm_resource id="prim_shared_storage:0" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="prim_shared_storage-0_monitor_0" operation="monitor" transition-key="6:0:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:7;6:0:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="2" crm_feature_set="1.0.9" rc-code="7" op-status="0" interval="0" op-digest="f05e197b6b37f94c2a3977cc41e66556"/>
           </lrm_resource>
           <lrm_resource id="shared_filesystem" type="Filesystem" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="shared_filesystem_monitor_0" operation="monitor" transition-key="7:0:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:7;7:0:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="3" crm_feature_set="1.0.9" rc-code="7" op-status="0" interval="0" op-digest="5858fd8b1fcc9107c71c24186f95f937"/>
           </lrm_resource>
           <lrm_resource id="prim_shared_storage:1" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="prim_shared_storage-1_monitor_0" operation="monitor" transition-key="5:1:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:7;5:1:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="4" crm_feature_set="1.0.9" rc-code="7" op-status="0" interval="0" op-digest="f05e197b6b37f94c2a3977cc41e66556"/>
             <lrm_rsc_op id="prim_shared_storage-1_start_0" operation="start" transition-key="6:2:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:0;6:2:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="5" crm_feature_set="1.0.9" rc-code="0" op-status="0" interval="0" op-digest="f05e197b6b37f94c2a3977cc41e66556"/>
             <lrm_rsc_op id="prim_shared_storage-1_post_notify_start_0" operation="notify" transition-key="42:2:65315fa6-aeaf-48f4-88b1-441ca298582a" transition-magic="0:0;42:2:65315fa6-aeaf-48f4-88b1-441ca298582a" call-id="6" crm_feature_set="1.0.9" rc-code="0" op-status="0" interval="0" op-digest="f05e197b6b37f94c2a3977cc41e66556"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="cd34ed93-fa5d-4092-a618-ac6349351d13">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="master-prim_shared_storage:1" value="75"/></instance_attributes>
       </transient_attributes>
     </node_state>
   </status>
 </cib>
